src/library/scala/runtime/LambdaDeserializer.scala

```diff
@@ -69,7 +71,15 @@ object LambdaDeserializer {
 
       // Lookup the implementation method
       val implMethod: MethodHandle = try {
-        findMember(lookup, getImplMethodKind, implClass, getImplMethodName, implMethodSig)
+        if (targetMethodMap != null) {
```

Comment:
(The caller has to provide the `Lookup`, so there isn't an exploit, but minimizing surface area is good practice.)